src/main/java/wolox/training/models/Book.java

```diff
@@ -67,6 +68,18 @@ public Book() {
         //Constructor for JPA
     }
 
+    public Book(BookInfoDTO bookInfoDTO) {
```

Comment:
This is fine, but personally I would leave this mapper at BookInfoDto, I would do the toModel method inside the dto.
I just think about leaving the model clean.

Reply:
Of course you could make model transformers to DTO and vice versa